pkg/kyma/cmd/install/cmd.go

```diff
@@ -193,6 +202,64 @@ func (cmd *command) installTiller() error {
 	if err != nil {
 		return err
 	}
+
+	return nil
+}
+
+func (cmd *command) configureHelm() error {
+	helmCmd := exec.Command("helm", "home")
+	helmHomeRaw, err := helmCmd.CombinedOutput()
+	if err != nil {
+		cmd.CurrentStep.LogError("Helm is not installed, will not configure it")
+		return nil
+	}
+	helmHome := strings.Replace(string(helmHomeRaw), "\n", "", -1)
+
+	secret, err := cmd.Kubectl().RunCmd("-n", "kyma-installer", "--ignore-not-found=false", "get", "secret", "helm-secret", "-o", "yaml")
+	if err != nil {
+		return err
+	}
+
+	cfg := make(map[interface{}]interface{})
+	err = yaml.Unmarshal([]byte(secret), &cfg)
+	if err != nil {
+		return err
+	}
+
+	data, ok := cfg["data"].(map[interface{}]interface{})
+	if !ok {
+		return fmt.Errorf("unable to get data from helm secret")
+	}
+
+	err = writeHelmFile(data, "global.helm.ca.crt", helmHome, "ca.pem")
+	if err != nil {
+		return err
+	}
+
+	err = writeHelmFile(data, "global.helm.tls.crt", helmHome, "cert.pem")
+	if err != nil {
+		return err
+	}
+
+	err = writeHelmFile(data, "global.helm.tls.key", helmHome, "key.pem")
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func writeHelmFile(data map[interface{}]interface{}, helmData string, helmHome string, filename string) error {
+	value, ok := data[helmData].(string)
+	if !ok {
+		return fmt.Errorf("unable to get %s from helm secret data", filename)
+	}
+	valueDecoded, err := base64.StdEncoding.DecodeString(value)
+
+	err = ioutil.WriteFile(filepath.Join(helmHome, filename), valueDecoded, 0777)
```

Comment:
@a-thaler Why would you give execution permission to the helm file?

Reply:
fixed